docs/proposals/organizations.md

```diff
@@ -0,0 +1,137 @@
+	Status ResourceQuotaStatus 
+}
+
+type ClusterResourceQuotaSpec struct {
+	Owner    *kapi.ObjectReference
+	Selector map[string]string 
+
+	kapi.ResourceQuotaSpec
+}
+```
+
+#### Limitations
+This approach is relatively easy to build, but it is limited because it doesn't reserve quota for a given project.  A more complicated approach would be to allow killing of "extra" resources and then provide a rate and ceiling like tc.  This would have an overall cap and allow "borrowing" of quota when resources are plentiful, but then selectively kill resources when resources are scarce to allow for project guarantees.  With enough effort, this sort of solution could be layered on top of kube by setting the ceiling for the existing quota controller and having a separate resource for the guarantee and a separate controller for the killer.
+
+I'm sure there will be complaints no matter what we do.
+
+
+## Project Owners
+Projects will have an label that indicates which Subject owns them.  If we don't allow modifications to project by default, then a cluster-admin can sort out project transfers.
+All project owners (both Users and Organizations) have a need to perform some actions scoped to them as an owner.  These will be represented as subresources under `oapi/v1/users/foo/<blah>` and `oapi/v1/organizations/foo/<blah>`.
+
+1. `resourcequotas` - Supports `list`.  This provides readonly access to all the `ClusterResourceQuotas` that apply to this owner.  Eventually we need to sort out `create` to allow additional subdivision by a user, but that would imply a need for `update` and `delete`.  `Update` and `delete`, should only be allowed on `resourcequotas` that the project owner has made.  That means we'd want some sort of "protected" flag or we could choose to model it as a different subresource backed by a different cluster-scope resource.
+1. `ownedprojects` - Supports `list`.  This provides readonly access to all the projects that you own.  We don't claim `projects`, because that's a likely spot to hang "all the projects you can see".
+
+
+
+## Organizations
+An Organization is a cluster scoped API object that references Users and has subresources for accessing and modifying Organization scoped OrgGroups.
+```go
+// Organization contains the set of Users that are members of the Organization.
+type Organization struct {
+	kapi.TypeMeta
+	kapi.ObjectMeta
+
+	// Owners is a list of Subjects that can administer this Organization.  By default a policy rule exists
+	// that allows all Users to inspect and modify OrgGroups of Organizations that they are an Admin for.
+	// Admins may NOT modify Organization membership.
+	Owners []kapi.ObjectReference
+
+	// Members is a list of Subjects that are members of the Organization.  Only members may be assigned to OrgGroups.
+	Members []kapi.ObjectReference
+}
+```
+
+The Organization has a list of admins.  Admins have the power to:
+ 1.  Reserve quota for particular projects.
```

Comment:
...based on the available `ClusterResourceQuota`? What can organization admins do if there are no `ClusterResourceQuota` allocated for their Organization?